test/prefer-complement.js

```diff
@@ -0,0 +1,53 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-complement';
+
+const ruleTester = avaRuleTester(test, {
+    env: {
+        es6: true
+    },
+    parserOptions: {
+        sourceType: 'module'
+    }
+});
+
+const instead = (composition, fn) => (composition === 'compose' ? `compose(not, ${fn})` : `pipe(${fn}, not)`);
+
+const error = (composition, fn) => ({
+  ruleId: 'prefer-complement',
+  message: `Instead of \`${instead(composition, fn)}\`, prefer \`complement(${fn})\``
+});
+
+ruleTester.run('prefer-complement', rule, {
+    valid: [
+        'complement(isEmpty)',
+        'complement(isNil)',
+        'propSatisfies(complement(isNil))',
```

Comment:
Can you add some valid test cases that use `pipe` and `compose` but don’t lead to a error? For example `compose(foo, bar)`.

Reply:
OK, will update PR.